locations/hours.py

```diff
@@ -133,10 +142,16 @@ def from_linked_data(self, linked_data, time_format="%H:%M"):
 
                     if "-" in days:
                         start_day, end_day = days.split("-")
+                        if start_day in DAY_MAPPING:
```

Comment:
You don't need to do this mapping array. `start_day[:2]` will give us the first 2 letters.

Reply:
Doh 🤦. Thanks, nice catch.

Reply:
Since were back to `LinkedDataParser.find_linked_data` and `LinkedDataParser.parse_ld` we can just do `LinkedDataParser.parse`, and still fix the opening hours later.

https://github.com/alltheplaces/alltheplaces/commit/3231e19179dcfe6aa8c5669d944e02eb6e8705d2 fixed this, before we had to remove the bad opening hours before `LinkedDataParser.parse_ld`.